### PR TITLE
Replace num_readouts and resign_threshold with flag versions

### PR DESCRIPTION
**Pull request by brilee**

### CODE REVIEW

File: strategies.py [hunk 1]
```diff
@@ -86,7 +93,8 @@ def __init__(self, network, seconds_per_move=5, simulations_per_move=0,
         self.root = None
         self.result = 0
         self.result_string = None
-        self.resign_threshold = -abs(resign_threshold)
+        self.resign_threshold = resign_threshold or FLAGS.resign_threshold
+        self.timed_match = timed_match
```
Comment (sethtroisi) [inline strategies.py]: consider timed_match or self.num_readouts == 0

File: cluster/selfplay/player_wrapper.sh [hunk 2]
```diff
@@ -23,8 +23,7 @@ echo bucket: $BUCKET_NAME
 echo board_size: $BOARD_SIZE
 
 python3 rl_loop.py selfplay \
-  --resign-threshold=0.99 \
-  --readouts=600
+  --num_readouts=600
```
Comment (amj) [inline cluster/selfplay/player_wrapper.sh]: Resign threshold still needs to go here.

File: strategies.py [hunk 3]
```diff
@@ -86,7 +93,8 @@ def __init__(self, network, seconds_per_move=5, simulations_per_move=0,
         self.root = None
         self.result = 0
         self.result_string = None
-        self.resign_threshold = -abs(resign_threshold)
+        self.resign_threshold = resign_threshold or FLAGS.resign_threshold
```
Comment (amj) [inline strategies.py]: it seems lke the flag value should take priority over the passed parameter, no?

Comment (brilee) [reply strategies.py]: fixed this by making it clearer that sometimes we pass in -1 to disable resign threshold. (in which case we override the flag)

File: strategies.py (hunk 1 above)
Comment (brilee) [reply strategies.py]: Added an assert to prevent cases where neither time / readout settings are configured.

File: strategies.py (hunk 3 above)
Comment (amj) [reply strategies.py]: ok, LGTM :)

**Comment by amj** [comment]
LGTM modulo the merge conflict

**Pull request merged**